test/embind/test_i64_binding.cpp

```diff
@@ -13,79 +13,76 @@
 using namespace emscripten;
 using namespace std;
 
-#define assert_js(X) assert(run_js(X))
+#define assert_js_eq(X, Y) run_js(string("const x = ") + X + ", y = " + Y + "; assert(x === y, `" + X + ": actual = ${typeof x} ${x}, expected = ${typeof y} ${y}`);")
 
 void test(string message)
 {
   cout << "test:\n" << message << "\n";
```

Comment:
How about just `printf("test: %s\n", message.c_str());`   (note that the first \n is removed so the output will need updating too).

Reply:
I don't mind, but that code was here before, seems out of scope of the PR?

Reply:
Fair enough.. I guess I couldn't tell if you were generally cleaning up this test or just updating it.

For example the `run_js` improvement below?  Is that needed or just a cleanup?  I'd be find with a cleanup PR landing first (which is why I normally do when I work on tests are crufty)

Reply:
Yeah, also fair... I just noticed that two functions were doing exactly same thing, so I removed one of them, but then, hey, it doesn't make sense for body to be that complicated and "accidentally" cleaned that up too.

I can change this line as well, np.